### ISSUE TITLE
Receiving warning about lenient param

### ISSUE THREAD

**Issue by dougmeredith** [octodns/octodns-ddns#49]
I get the warning "WARNING Manager provider DdnsSource does not accept lenient param". I've done a fair bit of research and a fair bit of playing with my configuration, but I can't get rid of it. 

My provider configuration is rather simple:

<code>
  public1:
    class: octodns_ddns.DdnsSource
    types: [A]
</code>

Am I doing something wrong?

**Comment by ross** [comment]
It's a warning that the API of the provider is using a deprecated set of arguments. There's already a PR to update it, https://github.com/octodns/octodns-ddns/pull/48, that warning will print until the PR is merged and an updated version is released. 

The existing version of the provider would not be compatible with octoDNS 2.0 once that releases, not any time soon.

TL;DR don't worry about it and update the provider once there's a newer version in a few weeks.

**Comment by dougmeredith** [comment]
Thanks for the information. :)
